packages/flutter/test/widgets/interactive_viewer_test.dart

```diff
@@ -519,7 +636,7 @@ void main() {
           home: Scaffold(
             body: Center(
               child: InteractiveViewer(
-                alignPanAxis: true,
+                alignPanAxis: PanAxis.aligned,
```

Comment:
I assume this test is also true for PanAxis.horizontal and vertical.  Probably no need to test those too though...